pkg/trtl/namespaces.go

```diff
@@ -1,10 +1,17 @@
 package trtl
 
+const (
+	NamespacePeers    = "peers"
+	NamespaceIndex    = "index"
+	NamespaceDefault  = "default"
+	NamespaceSequence = "sequence"
+)
+
 // Reserved namespaces that cannot be used by the caller since they are in use by trtl.
 // If necessary this can be moved to configuration in the future.
 var reservedNamespaces = map[string]struct{}{
-	"peers":    {},
-	"sequence": {},
-	"index":    {},
-	"default":  {}, // if the user does not specify a namespace
+	NamespacePeers:    {},
+	NamespaceSequence: {},
+	NamespaceIndex:    {},
+	NamespaceDefault:  {}, // if the user does not specify a namespace
 }
```

Comment:
Was this change related to the tests?

Reply:
I thought it would be easier to manage the code if we had these strings as constants.